src/api/users/post.js

```diff
@@ -0,0 +1,17 @@
+import { auth } from '../../firebase/index';
+
+export default function (company_email, company_password, company_name, company_photo_url) {
+  return new Promise((resolve, reject) => {
+    auth.createUserWithEmailAndPassword(company_email, company_password)
+    .then(userData => {
+      userData.user.updateProfile({
+        displayName: company_name,
+        photoURL: company_photo_url,
+      }).then(() => {
+        auth.currentUser.sendEmailVerification();
+        resolve(userData);
+      }); 
+    },
+    err => reject(err))
+  });
+}
```

Comment:
Since we are in JS, and the regular convention in this language is to use camelCase, what do you think of using it in this code?

```suggestion
export default function (companyEmail, companyPassword, companyName, companyPhotoUrl) {
  return new Promise((resolve, reject) => {
    auth.createUserWithEmailAndPassword(companyEmail, companyPassword)
    .then(userData => {
      userData.user.updateProfile({
        displayName: companyName,
        photoURL: companyPhotoUrl,
      }).then(() => {
        auth.currentUser.sendEmailVerification();
        resolve(userData);
      }); 
    },
    err => reject(err))
  });
}
```

Reply:
https://github.com/anonymouslyreportingtool/webapp/pull/5/commits/48b5824e3071be404d860863ce8b1184888eb596